webapp/channels/src/components/integrations/abstract_outgoing_webhook.tsx

```diff
@@ -171,67 +185,73 @@ export default class AbstractOutgoingWebhook extends React.PureComponent {
             team_id: this.props.team.id,
             channel_id: this.state.channelId,
             trigger_words: triggerWords,
-            trigger_when: parseInt(this.state.triggerWhen, 10),
+            trigger_when: this.state.triggerWhen,
```

Comment:
`Note:` See line 238 below to see why this was removed. It seems like the event handler was passing a `string`. We're expecting `triggerWhen` to be a `number`.